model-dmdocument/src/main/java/gov/nasa/pds/model/plugin/AbstractGenerationPlugin.java

```diff
@@ -67,7 +63,7 @@ public void execute() throws MojoExecutionException {
       getLog().info("Writing artifacts from those sources to ‘" + this.target + "’");
       File d = new File(this.target);
       d.mkdirs();
-      DMDocument.outputDirPath = d.toString() + "/";
+      // DMDocument.outputDirPath = d.toString() + "/";
```

Comment:
My rule when commenting out code:

- Add an explanation _why_ it's commented out
- Can't come up with an explanation? Then just _delete_! 😁